miner/worker.go

```diff
@@ -200,6 +200,14 @@ func (miner *Miner) prepareWork(genParams *generateParams) (*environment, error)
 		vmenv := vm.NewEVM(context, vm.TxContext{}, env.state, miner.chainConfig, vm.Config{})
 		core.ProcessBeaconBlockRoot(*header.ParentBeaconRoot, vmenv, env.state)
 	}
+	if miner.chainConfig.IsPrague(header.Number, header.Time) {
```

Comment:
Technically 2935 hasn't been CFId, so I suggest adding a `IsEIP2935` rule

Reply:
It's CFI now, so I think it's okay in `IsPrague`